scripts/build_against_electron.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+source ~/.nvm/nvm.sh
+
+set -e -u
+
+export DISPLAY=":99.0"
+GYP_ARGS="--runtime=electron --target=${ELECTRON_VERSION} --dist-url=https://atom.io/download/electron"
+NPM_BIN_DIR="$(npm bin -g 2>/dev/null)"
+
+function publish() {
+    if [[ ${PUBLISHABLE:-false} == true ]] && [[ ${COMMIT_MESSAGE} =~ "[publish binary]" ]]; then
+        node-pre-gyp package $GYP_ARGS
+        node-pre-gyp publish $GYP_ARGS
+        node-pre-gyp info $GYP_ARGS
+    fi
+}
+
+function electron_pretest() {
+    npm install -g electron@${ELECTRON_VERSION}
+    npm install -g electron-mocha
+    sh -e /etc/init.d/xvfb start
+    sleep 3
+}
+
+function electron_test() {
+    "$NPM_BIN_DIR"/electron test/support/createdb-electron.js
+    "$NPM_BIN_DIR"/electron-mocha -R spec --timeout 480000
+}
+
+# test installing from source
+npm install --build-from-source  --clang=1 $GYP_ARGS
+
+# TODO: remove me -start
```

Comment:
remove this tho - I've been leeching off your CI to make my own node-sqlite3 bins.